config/locales/client.en.yml

```diff
@@ -326,7 +326,9 @@ en:
 
   undecided_panel:
     aria_label: 'Undecided members'
-    show_undecided: Show undecided members
+    show: Show
+    remind: Remind
```

Comment:
Show & Remind feel like different order things. I'ma ponderin'. Leaning towards sticking it in the Options menu. Super-user feature. Could be fun to advertise it to coordinators via Intercom in-app message